mtags/src/main/scala/scala/meta/internal/mtags/ScalaToplevelMtags.scala

```diff
@@ -688,7 +688,6 @@ class ScalaToplevelMtags(
       case WHITESPACE =>
         nextIsNL()
       case COMMENT =>
-        scanner.skipComment()
```

Comment:
Why this change?

Reply:
The method got removed and it's not necessary since getting the next token will also skip any comments